hw/ip/rv_core_ibex/data/rv_core_ibex.prj.hjson

```diff
@@ -10,6 +10,7 @@
     version:            "1.0",
     life_stage:         "L1",
     design_stage:       "D2S",
-    verification_stage: "V1",
+    verification_stage: "N/A",
```

Comment:
I think the status would be clearer if we marked this as V2S and then still have checklist headers for each of the verification stages.

For example:

```
### V1

The V1 checklist may be found in the [Ibex documentation](https://ibexcore.readthedocs.io/en/latest/03_reference/verification_stages.html#v1-checklist).

### V2

The V2 checklist may be found in the [Ibex documentation](https://ibexcore.readthedocs.io/en/latest/03_reference/verification_stages.html#v2-checklist).

### V2S

The V2S checklist may be found in the [Ibex documentation](https://ibexcore.readthedocs.io/en/latest/03_reference/verification_stages.html#v2s-checklist).
```